README.md

```diff
@@ -19,12 +19,12 @@ In the European Union, this extension's fields correspond to [eForms BG-711 (Con
       "hasElectronicOrdering": false,
       "electronicInvoicingPolicy": "required",
       "reservedExecution": true,
-      "performanceTerms": "A set of KPIs will be developed for this contract and the successful contractor will be measured against these for the duration of the contract. Please refer to briefing document for further details.",
+      "performanceTerms": "A set of KPIs will be developed for this contract and the successful supplier will be measured against these for the duration of the contract. Please refer to briefing document for further details.",
```

Comment:
I think here it's more natural to have "successful bidder".

Reply:
Isn't "tenderer" the preferred term?

Reply:
Yes, you are correct :)